docs/docusaurus/multitenancy/access-control.md

```diff
@@ -31,7 +31,7 @@ Managed by DevOps and SRE teams.
 
 ### Management Account
 
-Management Account is special account that allows to manage the PRISM platform.
+Management Account is special account that allows to manage the Cloud Agent.
```

Comment:
In other instance, "PRISM platform" was changed to "Identus platform" but here, it is "Cloud Agent". Please double check.